go.mod

```diff
@@ -55,6 +55,7 @@ require (
 	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
 	golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae
 	google.golang.org/grpc v1.30.0
+	google.golang.org/grpc/examples v0.0.0-20200716233830-6dc7938fe875 // indirect
```

Comment:
Can we remove this?

Reply:
I unsure why it was added in the first place, my pattern to update dependencies was:
`go clean -modcache && go mod tidy && go mod vendor`, so it was added automatically by `go mod tidy`.

Reply:
If you switch to go 1.14, manually remove the line and run `make vendor` then the change should persist. :)

Reply:
Done, thanks for the tip!